Gemfile

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
 gemspec
+
+gem 'octokit', github: 'soudy/octokit.rb', branch: 'add-review-preview'
```

Comment:
Yes it should, but `spec.add_runtime_dependency` can not specify gem via github repository.
So I have to place it in `Gemfile` until the new version of octokit  is released to rubygems, which supports review API.